grafana/overlays/nerc-ocp-obs/externalsecrets/oauth-client-secret.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: oauth-client-secret
+  namespace: grafana
+spec:
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: oauth-client-secret
+  data:
+  - secretKey: GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET
+    remoteRef:
+      key: nerc/nerc-ocp-obs/dex/dex-clients
+      property: GRAFANA_SECRET
+
+  - secretKey: GF_AUTH_GENERIC_TLSCACERT
+    remoteRef:
+      key: nerc/nerc-ocp-obs/dex/grafanas
+      property: GF_TLSCACERT
```

Comment:
Could we simplify things by renaming the keys in vault to match the required secret key names and then using [`dataFrom`](https://github.com/OCP-on-NERC/nerc-ocp-config/blob/main/cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/externalsecret.yaml#L115) to pull them all in?

Reply:
yes, great idea